packages/pigeon/example/app/macos/Runner/MainFlutterWindow.swift

```diff
@@ -24,6 +24,12 @@ private class PigeonApiImplementation: ExampleHostApi {
     }
     completion(.success(true))
   }
+
+  /// Unlike implementations on other platforms, this function does not throw any exceptions
+  /// because the `@Async(type: AsyncType.await(isSwiftThrows: false))` annotation was specified.
```

Comment:
is there an example where isSwiftThrows = true?

Reply:
Not yet. Will add.

Reply:
Have added example